### DISCUSSION TITLE
Nano or tetra

### DISCUSSION THREAD

**Discussion by richoffpints** [xchwarze/wifi-pineapple-cloner#15] [Q&A] [answered]
Is there a way to only down load the nano firm ware or files and a way to only download the tetra files, if u dont mind showing or putting to gether a small how to for both, i looked already and i see the diffrnce in files, but i dont see the way to like, do it for one or the other

**Comment by xchwarze** [comment, marked as answer]
There are no "nano files" as it is the same software running on two quite different hardwares.
The differences between the two are mainly the drivers for the platforms and some minor changes in what they have installed by default to save space.

**Comment by medudder** [comment]
Hi!  I have a pineapple Nano.  I have to admit that my mangoapple works better.   VERY nice job!  Any chance that your firmware mode could work on an original Nano? thx
